python/ray/experimental/serve/global_state.py

```diff
@@ -50,6 +52,9 @@ def __init__(self):
         #  use random/available port in a pre-defined port range. TODO(simon)
         self.http_address = ""
 
+        #: Metric monitor handle
```

Comment:
Why is there a `:` after the `#`?

Reply:
This is sphinx's attribute notation
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute

Reply:
Ah ok I see!